osbs/utils.py

```diff
@@ -567,103 +566,6 @@ def sanitize_version(version):
     return cleaned_version
 
 
-class Labels(object):
```

Comment:
Since we would need to change atomic-reactor as well, I think doing the move once, and not twice makes a lot of sense. Compared to my current version, what if we did:

Move utils.py: => utils/__init__.py
Move label_utils.py => utils/labels.py (I think label*s* because the only thing in it is the Labels class)
Move repo_utils.py => utils/repo.py
Add 'from osbs.labels import Labels' to utils/__init__py so we don't have to change atomic reactor (at least for now)?

Does that sound like a plan?

Reply:
Hmm, actually, I guess I'd avoid moving repo_utils for now, since that would also require atomic-reactor changes - just leave it as osbs.repo_utils for now.

Reply:
Sounds good to me :+1:

Reply:
Sound good!